### DISCUSSION TITLE
How do I hide steamworks logs?

### DISCUSSION THREAD

**Discussion by doonv** [Thinkofname/steamworks-rs#176] [Q&A]
When I create a `Client` with `Client::init_app`, A bunch of logs get printed to stdout:
```
tid(56009) burning pthread_key_t == 0 so we never use it
[S_API] SteamAPI_Init(): Loaded '/home/doonv/.local/share/Steam/linux64/steamclient.so' OK.
Setting breakpad minidump AppID = 480
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  <insert steam id here> [API loaded no]
```
How do I hide the logs?

**Comment by Noxime** [comment]
These messages come from Steamworks SDK itself, and I am not quite sure how to disable the printing. I was not able to find anything with a quick Google search
